great_expectations/agent/actions/run_column_descriptive_metrics_action.py

```diff
@@ -34,10 +34,11 @@ def run(self, event: RunColumnDescriptiveMetricsEvent, id: str) -> ActionResult:
 
         self._metric_repository.add_metric_run(metric_run)
 
+        # TODO: What should resource_id be? Likely the returned id for the metric run from the rest API call.
```

Comment:
curious as to how/why this would be different from the `metric_run.id`. More of a question for my understanding, rather than a question for the PR

Reply:
This is a great question. After further thinking and looking more at the agent implementation I think it is the `metric_run.id`. I'll be cleaning this up in upcoming PRs on the agent action.